docs/network-flow-visibility.md

```diff
@@ -43,6 +44,45 @@ For visualizing the network flows, Antrea monitors the flows in Linux conntrack
 module. These flows are converted to flow records, and then flow records are post-processed
 before they are sent to the configured external flow collector.
 
+## Getting Started
+
+To get started, let's ensure you have installed Antrea with the Flow Exporter
+feature enabled by setting the following config parameter in `antrea.yaml`. 
```

Comment:
"the following `antrea-agent` config parameter in the Antrea deployment manifest"

In addition to antrea.yml, there are other YAML files too.